interfaces/fiveg_amf_n2/v0/schema.py

```diff
@@ -0,0 +1,32 @@
+"""This file defines the schemas for the provider and requirer sides of the `fiveg_amf_n2` interface.
+It exposes two interfaces.schema_base.DataBagSchema subclasses called:
+- ProviderSchema
+- RequirerSchema
+Examples:
+    ProviderSchema:
+        unit: <empty>
+        app: {"n2IpAddr": "192.168.251.6"}
+    RequirerSchema:
+        unit: <empty>
+        app:  <empty>
+"""
+
+from pydantic import BaseModel, IPvAnyAddress, Field
+
+from interface_tester.schema_base import DataBagSchema
+
+
+class MyProviderAppData(BaseModel):
+    url: IPvAnyAddress = Field(
```

Comment:
Here we use `url` and elsewhere we use `n2IpAddr`

Reply:
✅